### PR TITLE
Inception network support

### PR DESCRIPTION
**Pull request by Omarito2412**
In reference to #35 
Here I have added the required layers to support the network.

### CODE REVIEW

File: ide/static/js/data.js [hunk 1]
```diff
@@ -397,4 +397,30 @@ export default {
...
+      trg: ['LeftMiddle'],
+    },
+    params: {
+      bias_term: {
+        name: 'Bias term',
+        value: '',
+        type: 'select',
+        options: ['true', 'false'],
+        required: true,
+
```
Comment (dexter1691) [inline ide/static/js/data.js]: remove additional line

File: ide/static/bundle/bundle.js [hunk 2]
```diff
@@ -28362,6 +28362,32 @@
...
+	      trg: ['LeftMiddle']
+	    },
+	    params: {
+	      bias_term: {
+	        name: 'Bias term',
+	        value: '',
+	        type: 'select',
+	        options: ['true', 'false'],
+	        required: true
+
```
Comment (dexter1691) [inline ide/static/bundle/bundle.js]: remove additional line

**Comment by Omarito2412** [comment]
@dexter1691 I've modified both of the pull requests, feel free to merge.

**Comment by Omarito2412** [comment]
I've modified this PR too.

**Comment by Omarito2412** [comment]
To enable ResNet/ResNeXt support we needed to add the Eltwise layer on top of the BatchNorm and Scale layers that are already added in Inception, so I have added it and tested the Prototxt from #31 and it works now.

File: ide/utils/jsonToPrototxt.py [hunk 3]
```diff
@@ -345,6 +345,44 @@ def changeTopBlobName(layerId, newName):
...
+        elif (layerType == 'Eltwise'):
+            eltwise_param = {}
+            if layerParams['operation'] != '':
+                eltwise_param['operation'] = int(layerParams['operation'])
+            for ns in (ns_train,ns_test):
+                caffeLayer = get_iterable(L.Eltwise(
+                    *[ns[x] for x in blobNames[layerId]['bottom']],
+                    eltwise_param=eltwise_param
+                    ))
+                    #*([ns[x] for x in blobNames[layerId]['bottom']] + [ns.label])))
```
Comment (dexter1691) [inline ide/utils/jsonToPrototxt.py]: remove unnecessary comments.

Comment (Omarito2412) [inline ide/utils/jsonToPrototxt.py]: Done

**Comment by Omarito2412** [comment]
@dexter1691 I've fixed the conflicts on this one too

**Pull request merged**